ci/jenkins/README.md

```diff
@@ -273,3 +297,4 @@ updated with new code.
 ## Tips for Developer
 
 * [macro.yaml](jobs/macros.yaml): Use "{{}}" instead of "{}" in "builder-list-tests" and "builder-conformance".
+* While setting up the rancher testbed, delete the cattle-cluster-agent deployment and use cattle-node-agent because cluster-agent adds extra watchers for all the resources.
```

Comment:
Maybe add some more words like "Antrea-controller counts connected Antrea-agents from watcher connections. Extra watchers lead to wrong Antrea-agent number in AntreaControllerInfo CR."